api/app/src/external/commonwell/document/document-query.ts

```diff
@@ -1,40 +1,53 @@
-import { CommonwellError, DocumentQueryResponse } from "@metriport/commonwell-sdk";
-import { MedicalDataSource } from "../..";
-import { createOrUpdate } from "../../../command/medical/document/create-or-update";
+import { CommonwellError, DocumentQueryResponse, Document } from "@metriport/commonwell-sdk";
+import stream from "stream";
+import * as AWS from "aws-sdk";
 import { updateDocQueryStatus } from "../../../command/medical/document/document-query";
-import { DocumentReference } from "../../../domain/medical/document-reference";
-import { DocumentReferenceModel } from "../../../models/medical/document-reference";
 import { Patient } from "../../../models/medical/patient";
+import { Organization } from "../../../models/medical/organization";
+import { Facility } from "../../../models/medical/facility";
 import { capture } from "../../../shared/notifications";
 import { oid } from "../../../shared/oid";
 import { Util } from "../../../shared/util";
 import { makeCommonWellAPI, organizationQueryMeta } from "../api";
 import { getPatientData, PatientDataCommonwell } from "../patient-shared";
-import { DocumentWithLocation, getFileName, toDomain } from "./shared";
+import { downloadDocument } from "./document-download";
+import { upsertDocumentToFHIRServer } from "../../fhir/document/save-document-reference";
+import { toFHIR as toFHIRDocRef } from "./shared";
+import { DocumentWithFilename } from "./shared";
+import { Config } from "../../../shared/config";
+import { createS3FileName } from "../../../shared/external";
+import { debug } from "../../../shared/log";
+
+const s3client = new AWS.S3();
 
 export async function getDocuments({
   patient,
   facilityId,
 }: {
   patient: Patient;
   facilityId: string;
-}): Promise<DocumentReference[]> {
+}): Promise<void> {
   try {
-    const cwDocuments = await internalGetDocuments({ patient, facilityId });
+    const { organization, facility } = await getPatientData(patient, facilityId);
 
-    const documents = cwDocuments.map(toDomain(patient));
+    const cwDocuments = await internalGetDocuments({ patient, organization, facility });
 
-    // Commonwell does not consistently return the same ID for a given document, so we don't
-    // update CW's entries, we replace them
-    await DocumentReferenceModel.destroy({
-      where: {
-        cxId: patient.cxId,
-        patientId: patient.id,
-        source: MedicalDataSource.COMMONWELL,
-      },
+    if (Config.isSandbox()) {
+      return;
+    }
```

Comment:
Cant do upstream because we need to call this to complete polling, but ill put it at the top of the try catch

`await updateDocQueryStatus({ patient, status: "completed" });`